### ISSUE TITLE
Investigate impact of Deno moving its std library to JSR

### ISSUE THREAD

**Issue by MasterKale** [MasterKale/SimpleWebAuthn#571]
## Describe the issue

See https://deno.com/blog/std-on-jsr, this part makes me wonder if I have to start using JSR in here, which would have an impact on the rest of the project:

> deno.land/std will still be available indefinitely. All programs that depend on deno.land/std will keep working. Don’t worry!
> 
> However, going forward new features will be published to jsr.io/@std. deno.land/std will receive only critical updates, such as security patches.

**Comment by sachaw** [comment]
For those of us using Deno, it would be good if all of the packages could be published on JSR.
Would be nice in the future to see this project use JSR as the package source for NPM (using JSR's automatic transpiling)

**Comment by MasterKale** [comment]
> Would be nice in the future to see this project use JSR as the package source for NPM (using JSR's automatic transpiling)
> This could possibly remove the need to use DNT.

I am with you on this, JSR stands to greatly simplify my work of deploying this project to support as many runtimes as I target. The one thing that seems a little suboptimal for me right now is the need to use `npx jsr add` to install something, and remember to commit the generated .npmrc to help future `npm install` commands know where to pull in SimpleWebAuthn.

Is anyone on the NPM side of JS runtimes working to add first-class support for JSR package installations?

**Comment by MasterKale** [comment]
I'm going to add this conversation to this issue:

https://github.com/MasterKale/SimpleWebAuthn/discussions/542

tl;dr: _Technically_ **@simplewebauthn/browser** is available via https://deno.land/x/simplewebauthn@v11.0.0/packages/browser, but its code is not in a shape that Deno recognizes (no file extensions on relative imports, etc...) Could migrating/deploying this project to JSR make @simplewebauthn/browser available in Deno projects too?

**Comment by MasterKale** [comment]
If anyone's following this issue and is interested in testing out Deno support via JSR-hosted versions of v11.0.0, check out this discussion comment for more info:

https://github.com/MasterKale/SimpleWebAuthn/discussions/542#discussioncomment-11195351

**Comment by MasterKale** [comment]
SimpleWebAuthn will now enjoy first-class support for installation from JSR as well as from NPM as of v12.0.0 🎉 

See here for more info:

https://github.com/MasterKale/SimpleWebAuthn/releases/tag/v12.0.0
